src/rapids_dependency_file_generator/cli.py

```diff
@@ -51,35 +60,62 @@ def validate_args(argv):
         ],
     )
     codependent_args.add_argument(
+        "-m",
         "--matrix",
         help=(
             "String representing which matrix combination should be generated, "
             'such as `--matrix "cuda=11.5;arch=x86_64"`. May also be an empty string'
         ),
     )
 
+    parser.add_argument(
+        "--prepend-channel",
+        action="append",
+        default=[],
+        dest="prepend_channels",
+        help=(
+            "A string representing a conda channel to prepend to the list of "
+            "channels. This option is only valid with --output "
+            f"{OutputTypes.CONDA} or no --output. May be specified multiple times."
+        ),
+    )
     parser.add_argument(
         "--prepend-channels",
+        dest="prepend_channels_deprecated",
         help=(
             "A string representing a list of conda channels to prepend to the list of "
             "channels. Channels should be separated by a semicolon, such as "
             '`--prepend-channels "my_channel;my_other_channel"`. This option is '
-            f"only valid with --output {OutputTypes.CONDA} or no --output"
+            f"only valid with --output {OutputTypes.CONDA} or no --output. "
+            "DEPRECATED: Use --prepend-channel instead."
         ),
     )
 
     args = parser.parse_args(argv)
+
+    if args.file_key_deprecated:
+        warnings.warn(
+            "The use of --file_key is deprecated. Use -f or --file-key instead."
+        )
+    if not args.file_key:
+        args.file_key = args.file_key_deprecated
+
     dependent_arg_keys = ["file_key", "output", "matrix"]
     dependent_arg_values = [getattr(args, key) is None for key in dependent_arg_keys]
     if any(dependent_arg_values) and not all(dependent_arg_values):
         raise ValueError(
             "The following arguments must be used together:"
-            + "".join([f"\n  --{x}" for x in dependent_arg_keys])
+            + "".join([f"\n  {x}" for x in ["--file-key", "--output", "--matrix"]])
         )
 
+    if args.prepend_channels_deprecated:
```

Comment:
Raise if both are used.
```suggestion
    if args.prepend_channels_deprecated:
        if args.prepend_channels:
            raise ValueError("The --prepend-channels (deprecated) and --prepend-channel arguments cannot be specified together.")
```